internal/ir/ir.go

```diff
@@ -231,6 +232,19 @@ type Return struct {
 	Results []Value
 }
 
+// Struct instruction represents a class or struct.
+//
+// type   ExampleClass
+//   method(ExampleClass) exampleMethod(self)
+//
+// The Struct implements Value and Member interfaces.
+type Struct struct {
+	node
+	name    string      // Class name.
+	Fields  []Value     // Class attributes.
+	Methods []*Function // Class methods.
```

Comment:
Since in the IR we will always refer a class as struct this comments should rename "Class" to "Struct".

Reply:
Done!